src/tlo/analysis/probability_premature_death.py

```diff
@@ -0,0 +1,107 @@
+"""
+Read in the output files generated by analysis_scenarios
+generate life tables to estimate probability of premature death (defined as before age 70) for each run/draw
+produce summary statistics
+"""
+
+
+import datetime
+from pathlib import Path
+from typing import Dict, Tuple
+
+import pandas as pd
+
+from tlo.analysis.life_expectancy import _aggregate_person_years_by_age, _num_deaths_by_age_group
+from tlo.analysis.utils import get_scenario_info, summarize
+
+
+def _calculate_probability_of_dying_before_70(
```

Comment:
I would rename to `_calculate_probability_of_premature_death` and throughout 'soft-code' the "magic number of 70".

I would declare at the top of the script

`AGE_BEFORE_WHICH_DEATH_IS_DEFINED_AS_PREMATURE = 70`


and provide the reference to the paper that is cited in the issue.

That way we're super-clear about the origin of the magic number. And, if, we change our mind about the definition, it's a simple change.

Reply:
Done!